src/styles/field.scss

```diff
@@ -0,0 +1,10 @@
+.field {
+    width: 1000px;
+    height: 600px;
```

Comment:
used smart size instead px

Reply:
But for width I used %, because for desktops, the browser window is wider than the viewport (adding the width of the scrollbar), so if you set the width to 100vw for the element, it will go outside the html.